site/app/controllers/forum/ForumController.php

```diff
@@ -360,7 +360,13 @@ public function publishThread() {
             }
             else {
                 // Good Attachment
-                $result = $this->core->getQueries()->createThread($markdown, $current_user_id, $thread_title, $thread_post_content, $anon, $pinned, $thread_status, $hasGoodAttachment[0], $categories_ids, $lock_thread_date, $expiration, $announcement);
+                $attachment_name = "";
+                if ($hasGoodAttachment[0] == 1) {
+                    for ($i = 0; $i < count($_FILES['file_input']["name"]); $i++) {
+                        $attachment_name = $attachment_name . "\n" . basename($_FILES['file_input']["name"][$i]);
+                    }
+                }
+                $result = $this->core->getQueries()->createThread($markdown, $current_user_id, $thread_title, $thread_post_content, $anon, $pinned, $thread_status, $hasGoodAttachment[0], $attachment_name, $categories_ids, $lock_thread_date, $expiration, $announcement);
```

Comment:
This line is pretty long.  Perhaps something like this would be more readable:
```suggestion
                $result = $this->core->getQueries()->createThread(
                    $markdown,
                    $current_user_id,
                    $thread_title,
                    $thread_post_content,
                    $anon,
                    $pinned,
                    $thread_status,
                    $hasGoodAttachment[0],
                    $attachment_name,
                    $categories_ids,
                    $lock_thread_date,
                    $expiration,
                    $announcement
                );
```